javascript/HtmlEditorField.js

```diff
@@ -1158,8 +1158,17 @@ ss.editorWrappers['default'] = ss.editorWrappers.tinyMCE;
 				var urlField = this.getURLField(), container = this.closest('.CompositeField'), form = this.closest('form');
 
 				if (urlField.validate()) {
+
+					var val = urlField.val(), orig = val;
+
 					container.addClass('loading');
-					form.showFileView('http://' + urlField.val()).done(function() {
+
+					// add "http://" if a protocol is missing from the url
+					if (val.match(/^https?:\/\//i) == void 0) {
```

Comment:
Maybe this pattern should be `/^(https?:)?\/\//i` so that we can support scheme-less protocols as well? I know it's considered a bit of an anti-pattern now but some people may still want to use it?